Anchor.toml

```diff
@@ -6,14 +6,16 @@ cluster = "localnet"
 wallet = "~/.config/solana/id.json"
 
 [workspace]
-members = ["programs/invariant", "programs/staker"]
+members = ["programs/invariant",
+#  "programs/staker"
+ ]
 
 [programs.localnet]
-invariant = "HyaB3W9q6XdA5xwpU4XnSZV94htfmbmqJXZcEbRaJutt"
+invariant = "5KLfYtAYvUchvTg8jb27eYgeRXLjPPsa9DqwjJJuYGQJ"
 staker = "MJ6WF1tpEJ7Gk8ULqejDJapRfqBwBEp1dH5QvAgYxu9"
 
 [programs.devnet]
-invariant = "D8Xd5VFXJeANivc4LXEzYqiE8q2CGVbjym5JiynPCP6J"
+invariant = "5KLfYtAYvUchvTg8jb27eYgeRXLjPPsa9DqwjJJuYGQJ"
```

Comment:
Addresses should stay the same

Reply:
Before the update the devnet address were different than mainnet one. After the upgrade it shouldn't change anything tbh